src/transport.ts

```diff
@@ -1,70 +1,142 @@
 import Daily, {
```

Comment:
is this file a straight up copy from the transports repo? does that mean RN only supports daily as a transport? or is it that only daily has to be overridden so that the daily pieces are pulled from rn-daily-js?